$OpenBSD: patch-src_network_hpp,v 1.3 2014/11/29 06:46:33 kirby Exp $
Make sure netinet/in.h is included before SDL_net.h to
avoid redefining INADDR_ANY/NONE/BROADCAST in SDL header
--- src/network.hpp.orig	Sat Aug 17 17:19:31 2013
+++ src/network.hpp	Wed Jul 16 08:10:24 2014
@@ -22,6 +22,10 @@
 
 class config;
 
+#include <sys/types.h>
+#include <sys/socket.h>
+#include <netinet/in.h>
+
 #include "exceptions.hpp"
 #include "SDL_net.h"
 
